model_hub/model_hub/mmdetection/_trial.py

```diff
@@ -179,7 +176,7 @@ def train_batch(self, batch: Any, epoch_idx: int, batch_idx: int) -> Dict[str, t
         batch = {key: batch[key].data[0] for key in batch}
 
         losses = self.model(**batch)
-        loss, log_vars = self.model._parse_losses(losses)
+        loss, log_vars = self.model.module._parse_losses(losses)
```

Comment:
we're suppose to rely on passthrough here right?  since otherwise this wouldn't work for single slot training.